contrib/pacman/PKGBUILD

```diff
@@ -0,0 +1,37 @@
+pkgname='gollama'
+pkgdesc='Go manage your Ollama models'
+pkgver='1.27.6'
+pkgrel='1'
+arch=('x86_64')
+url='https://github.com/sammcj/gollama'
+license=('MIT')
+depends=('glibc')
+makedepends=('go')
+optdepends=('ollama: ollama support.')
+source=("${pkgname}-${pkgver}.tar.gz::https://github.com/sammcj/gollama/archive/refs/tags/v${pkgver}.tar.gz")
+b2sums=('2b2d77d52e321b86010739f0828b9de1f585c3ca86d3bc30e818d4d3e7b317676dd7fa526e349557eed7eae9630f8fc2ce50b4fbd524c329b4935ab00d6af24a')
```

Comment:
Does this somehow automatically get updated?

Reply:
It doesn't. Typically the b2sums are re-calculated via the `updpkgsums` command. This would be more useful published to the [AUR](https://aur.archlinux.org/) as-is. I wonder if a github action post release hook could generate this dynamically.

Alternatively we could loosen the restrictions for the package and use a `-git` package instead. Update incoming.